.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -17,8 +17,14 @@ Each of the questions and sections below start with multiple hash symbols (#). P
 
 ### Actual behavior:
 
+#### Actual NVDA speech/braille sample:
```

Comment:
What if this is an instruction under the heading instead?

```suggestion
### Actual behavior:
<!--
Use "Speak command keys" (NVDA+4) and speech viewer to copy and paste here. 
Use braille viewer to copy and paste here.
You may additionally include an explanation.
-->
```

Reply:
I think these instructions are more likely to be ignored than an explicit field, but this might be a better solution as copied speech/braille is not always relevant (as per https://github.com/nvaccess/nvda/pull/13905#discussion_r921711076).

Fixing braille viewer copy/paste sounds like an important thing to do as well.